assets/js/blocks/featured-product/style.scss

```diff
@@ -49,10 +55,11 @@
 	.wc-block-featured-product__variation,
 	.wc-block-featured-product__description,
 	.wc-block-featured-product__price {
-		color: $white;
 		line-height: 1.25;
 		margin-bottom: 0;
 		text-align: center;
+		color: inherit;
+		font-size: inherit;
```

Comment:
Personally, I think the `.wc-block-featured-product__title` shouldn't have `font-size: inherit;`. That changes the default design:

| `trunk` | This branch |
| --- | --- |
| ![imatge](https://user-images.githubusercontent.com/3616980/151779634-74485345-6180-44bf-91d1-a17b7ccc229f.png) | ![imatge](https://user-images.githubusercontent.com/3616980/151779682-ae354393-c94d-4187-bb95-2cef7ff33b9d.png) |

What do you think?

Reply:
You are right! 
This is a bug that affected `Featured Category` too :(  (already merged in the trunk), but this bug is replicable only when the site has a block theme enabled.

The Global Styles API doesn't support different font-size options, so we can:
- disable the font size option.
- font size option will impact only the description (it doesn't make sense IMHO).

Reply:
In the end, I followed the same approach that we adopted in #5682: the user can't customize font-size